### DISCUSSION TITLE
I'm not able to push to GH

### DISCUSSION THREAD

**Discussion by salochara** [joemasilotti/railsdevs.com#172] [Q&A] [answered]
Hi there! 
Hope everyone is doing well. 

I'm having an issue when pushing changes to `origin`.
I keep getting the same error when pushing changes to GH.
```
ERROR: Permission to joemasilotti/railsdevs.com.git denied to salochara.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I've tried it all (I believe). I've followed the tips in this [link](https://stackoverflow.com/questions/25927914/git-error-please-make-sure-you-have-the-correct-access-rights-and-the-reposito) but had no luck. 

I've talked to a colleague and he believes it might be that I do not have permissions in this repo to push, although this is a public repo.
Is there an additional step we need to do so we can push changes? @joemasilotti 

TIA and happy holidays everyone!

**Comment by joemasilotti** [comment, marked as answer]
Hey @salochara! The repo is public, but only for reading. You won't be able to push directly to it.

However, to submit code changes/requests, fork the repo and push to your fork. Then you will be able to submit a pull request that I can review and merge in to this one!

Here's some more info from GitHub on forks and pull requests: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

Let me know if that doesn't work or you need help!

**Reply by salochara** [reply]
Thank you! That was it! 
I've created my PR 😁
